x-pack/test/security_solution_api_integration/config/services/security_solution_ess_utils.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../../ftr_provider_context';
+import { SecuritySolutionUtils } from './types';
+
+export async function SecuritySolutionESSUtils({
+  getService,
+}: FtrProviderContext): Promise<SecuritySolutionUtils> {
+  const config = getService('config');
+
+  return {
+    getUsername: (_role?: string) =>
```

Comment:
The `_role` param seems to be unused

Reply:
Yes, there aren't roles on ESS. But we need the same method signature so that it can be called from ESS and Serverless. I used the `_` before the parameter to sign that it is unused.